.github/workflows/test_pull_requests.yml

```
@@ -162,7 +179,22 @@ jobs:
           NUMPY_EXPERIMENTAL_ARRAY_FUNCTION: ${{ matrix.MIN_REQ || 1 }}
           PYVISTA_OFF_SCREEN: True
           MIN_REQ: ${{ matrix.MIN_REQ }}
-
+      # this steps display the cached folder size for debug and
+      # remove the biggest contenders.
+      - name: Display Cache size, and remove largest files from cache.
+        if: runner.os == 'Linux'
+        run: |
+          echo "Biggest files in site-packages"
+          du -s /tmp/.tox/*py*/lib/py*/site-packages/* | sort -n | tail -n 5
+          echo "Biggest files in tox cache"
+          du -hs /tmp/.tox/*
+          echo "Cleaning a few of the biggest cache"
```

```suggestion
echo "Cleaning a few of the biggest cached items (semgrep,torch)
```

Initially, I got confused thinking you were always clearing out the largest 5 items.  You can clearly see that's not going on in the logs already, but modifying the comment might clarify.

---

Thanks !